src/test/java/org/opensearch/ad/caching/PriorityCacheTests.java

```diff
@@ -656,24 +696,4 @@ public void testGetNoPriorityUpdate() {
         assertTrue(null != entityCache.get(entity1.getModelId(detectorId).get(), detector));
         assertTrue(null == entityCache.get(entity2.getModelId(detectorId).get(), detector));
     }
-
-    public void testRemoveEntityModel() {
-        for (int i = 0; i < 3; i++) {
-            // bypass doorkeeper
-            entityCache.get(entity2.getModelId(detectorId).get(), detector);
-        }
-
-        // fill in dedicated cache
-        entityCache.hostIfPossible(detector, modelState2);
-
-        assertTrue(null != entityCache.get(entity2.getModelId(detectorId).get(), detector));
-
-        entityCache.removeEntityModel(detectorId, entity2.getModelId(detectorId).get());
-
-        assertTrue(null == entityCache.get(entity2.getModelId(detectorId).get(), detector));
-
-        verify(checkpoint, times(1)).deleteModelCheckpoint(eq(entity2.getModelId(detectorId).get()), any());
-        verify(checkpointWriteQueue, never()).write(any(), anyBoolean(), any());
-    }
```

Comment:
Why are these changes removed, I dont understand what they do (yet) but shouldn't we test them?

Reply:
Same as above